python/paddle/fluid/layers/io.py

```diff
@@ -448,6 +449,88 @@ def random_data_generator(low, high, shapes, lod_levels, for_parallel=True):
     return monkey_patch_reader_methods(main_prog_var)
 
 
+def py_reader(capacity, shapes, lod_levels, dtypes, for_parallel=True):
+    """
+    Create a reader and blocking queue for data feeding in Python
+
+    This layer returns a Reader Variable and a BlockingQueue.
```

Comment:
This API could may not be displayed well in sphinx.

See https://github.com/PaddlePaddle/FluidDoc